packages/commercetools/theme/assets/ui-state.js

```diff
@@ -0,0 +1,17 @@
+import Vue from 'vue'
+import VueCompositionApi, { reactive, ref, computed } from '@vue/composition-api'
+
+Vue.use(VueCompositionApi);
```

Comment:
It's already registered in nuxt module,so you can just use it as it is

Reply:
It doesnt work without:

<img width="1679" alt="Screen Shot 2019-12-24 at 9 48 53 AM" src="https://user-images.githubusercontent.com/10549495/71401459-9fd55500-2632-11ea-94b6-eee302977aa6.png">

Reply:
This is really weird since the plugin is registered already (setup function requires it to run). I think it needs some further investigation